network_simulator/host.py

```diff
@@ -2,6 +2,11 @@
 """
 
 from device import *
+from event import *
+from flow import *
+from link import *
+from statistics import *
+import bisect as b
```

Comment:
"bisect" is a standard Python module and so its import should come before "from device import *" etc (leave a blank line in between too). Format for imports is 1) standard Python imports, (line break), 2) third-party imports, (line break), 3) imports related to our own project.

Also, if you import bisect as b, you should refer to it as "b" below. I see some instances of bisect.bisect_left etc.